### ISSUE TITLE
Invalid authorization residue after deleting

### ISSUE THREAD

**Issue by unilinu** [fleetboard-io/fleetboard#117]
## Bug
re-install fleetboard agent by helm delete then helm install: 

```sh
W1210 02:21:44.420189       1 reflector.go:535] pkg/mod/k8s.io/client-go@v0.28.4/tools/cache/reflector.go:229: failed to list *v1.EndpointSlice: Unauthorized
E1210 02:21:44.420226       1 reflector.go:147] pkg/mod/k8s.io/client-go@v0.28.4/tools/cache/reflector.go:229: Failed to watch *v1.EndpointSlice: failed to list *v1.EndpointSlice: Unauthorized
E1210 02:21:49.711172       1 config.go:174] failed to list peers from hub cluster, loop next Unauthorized
```

## Fix

manually delete fleetboard secrets in child cluster, then re-install again.

`kubectl delete secrets -n fleetboard-system fleetboard`

**Comment by lmxia** [comment]
You are right.

That is because `fleeltboard-system` namespace is not deleted when fleetboard chart deleted.

We may add `fleeltboard-system ` namespace in helm chart template and remove ` create-namespace`  params in helm install commands. So it will be deleted automatically.

Could you fix it?

**Comment by unilinu** [comment]
Yes

**Comment by unilinu** [comment]
Fixed by [fleetboard-charts #10](https://github.com/fleetboard-io/fleetboard-charts/pull/10)
